lms/templates/emails/credit_eligibility_email.txt

```diff
@@ -0,0 +1,8 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hi {name},").format(name=full_name)}
+
+${_("Congratulation! You are eligible to get credit for the {course}").format(course=course_name)}
```

Comment:
I think there might be a small typo here: maybe just "You are eligible to get credit for {course}" (instead of "the {course}")